src/main/pages/che-7/extensions/assembly_telemetry.adoc

```diff
@@ -0,0 +1,665 @@
+        <artifactId>httpclient</artifactId>
+        <version>4.5.12</version>
+    </dependency>
+----
+
+We also add the Apache HTTP components library to send HTTP requests.
+
+Consult the link:https://github.com/che-incubator/che-workspace-telemetry-client/packages[GitHub packages] for the latest version and maven coordinates of `backend-base`.  GitHub packages require a link:https://help.github.com/en/packages/publishing-and-managing-packages/about-github-packages#about-tokens[personal access token] with `read:packages` permissions in order to download the {prod-short} telemetry libraries.  Create a personal access token and copy the token value.
+
+We will then create a `settings.xml` in the repository root and add the coordinates and token to the `che-incubator` packages:
+
+.settings.xml
+[source,xml]
+----
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+                      http://maven.apache.org/xsd/settings-1.0.0.xsd">
+   <servers>
+      <server>
+         <id>che-incubator</id>
+         <username>${env.GITHUB_USERNAME}</username>
+         <password>${env.GITHUB_TOKEN}</password>
+      </server>
+   </servers>
+
+   <profiles>
+      <profile>
+         <id>github</id>
+         <activation>
+            <activeByDefault>true</activeByDefault>
+         </activation>
+         <repositories>
+            <repository>
+               <id>central</id>
+               <url>https://repo1.maven.org/maven2</url>
+               <releases><enabled>true</enabled></releases>
+               <snapshots><enabled>false</enabled></snapshots>
+            </repository>
+            <repository>
+               <id>che-incubator</id>
+               <name>GitHub navikt Apache Maven Packages</name>
+               <url>https://maven.pkg.github.com/che-incubator/che-workspace-telemetry-client</url>
+            </repository>
+         </repositories>
+      </profile>
+   </profiles>
+</settings>
+----
+
+This file will be used when packaging our application in a container.  If you are running this locally, you can add the information to your personal `settings.xml` file.
+
+== Running the application
+
+The easiest way to run and test the application is in a {prod-short} workspace.  Run the application with
+
+```
+mvn quarkus:dev -Dquarkus.http.port=${CHE_WORKSPACE_TELEMETRY_BACKEND_PORT}
+```
+
+Note that if {prod-short} is secured using a self-signed certificate, you will need to add the certificate to a trust store and mount it into the workspace.  You will also need to add the java system property `-Djavax.net.ssl.trustStore=/path/to/trustStore` to the `mvn` command.  
+For example, assuming your trust store is located in `$JAVA_HOME/jre/lib/security/cacerts`:
+
+`keytool -import -alias self-signed-certificate -file path/to/self-signed-certificate -keystore $JAVA_HOME/jre/lib/security/cacerts`
```

Comment:
my understanding that this does not work atm due to https://github.com/quarkusio/quarkus/issues/9713

Reply:
You will still need to do that even when building a non-native container